amqphandler/amqphandler.go

```diff
@@ -234,6 +234,7 @@ func (handler *AmqpHandler) SendUnitStatus(unitStatus cloudprotocol.UnitStatus)
 	handler.Lock()
 	defer handler.Unlock()
 
+	unitStatus.MessageType = cloudprotocol.UnitStatusMessageType
 	return handler.scheduleMessage(cloudprotocol.UnitStatusMessageType, unitStatus, false)
```

Comment:
Looks like now it is not needed to pass message type to `scheduleMessage` function. Also remove message type  from message header in aos_common